src/clue.js

```diff
@@ -1,94 +1,110 @@
-occupation:   Actor
+// Characters Collection
+var charactersArray = [
 
-mrsPeacock
-first_name:   Eleanor
-last_name:    Peacock
-color:        blue
-description:  She is from a wealthy family and uses her status and money to earn popularity
-age:          36
-image:        https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-occupation:   Socialité
+  drOrchid = {
+  first_name:   "Doctor",
+  last_name:    "Orchid",
+  color:        "white",
+  description:  "PhD in plant toxicology. Adopted daughter of Mr. Boddy",
+  age:          26,
+  image:        "http://www.radiotimes.com/uploads/images/Original/111967.jpg",
+  occupation:   "Scientist",
+  },
 
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-occupation:   Retired Football player
+  profPlum = {
+  first_name:   "Victor",
+  last_name:    "Plum",
+  color:        "purple",
+  description:  "Billionare video game designer",
+  age:          22,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+  occupation:   "Designer",
+  },
 
-// Weapons
+  missScarlet = {
+  first_name:   "Kasandra",
+  last_name:    "Scarlet",
+  color:        "red",
+  description:  "She is an A-list movie star with a dark past",
+  age:          31,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+  occupation:   "Actor",
+  },
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+  mrsPeacock = {
+  first_name:   "Eleanor",
+  last_name:    "Peacock",
+  color:        "blue",
+  description:  "She is from a wealthy family and uses her status and money to earn popularity",
+  age:          36,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+  occupation:   "Socialité",
+  },
 
-// Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+  mrMustard = {
+  first_name:   "Jack",
+  last_name:    "Mustard",
+  color:        "yellow",
+  description:  "He is a former football player who tries to get by on his former glory",
+  age:          62,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation:   "Retired Football player"
+  }
+];
 
-// Characters Collection
-var charactersArray = [];
+var randomCard;
+function randomSelector (para) {
```

Comment:
What does para means? Naming things in code is a big topic and very important! Try to name things so that other developers don't have to think what it is